stock_picking_portal/data/demo.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<odoo noupdate='1'>
+    <record id="outgoing_shipment_stock_picking_portal" model="stock.picking">
+            <field name="picking_type_id" ref="stock.picking_type_out" />
+            <field name="origin">stock_picking_portal outgoing shipment</field>
+            <field name="user_id" />
+            <field name="partner_id" ref="base.partner_admin" />
+            <field name="scheduled_date" eval="DateTime.today()" />
+            <field name="location_id" ref="stock.stock_location_stock" />
+            <field name="location_dest_id" ref="stock.stock_location_customers" />
+            <field
+            name="move_ids"
+            model="stock.move"
+            eval="[(0, 0, {
+                'name': obj().env.ref('product.product_product_27').name,
+                'product_id': ref('product.product_product_27'),
+                'product_uom': ref('uom.product_uom_unit'),
+                'product_uom_qty': 15.0,
+                'picking_type_id': ref('stock.picking_type_out'),
+                'location_id': ref('stock.stock_location_stock'),
+                'location_dest_id': ref('stock.stock_location_customers'),
+            })]"
+        />
+        </record>
+
+    <record id="incomming_stock_picking_portal" model="stock.picking">
```

Comment:
Something is not working here.
It does not show on the portal. Trying to validate this returns in an error:

Incompatible companies on records:
- 'WH/IN/00006' belongs to company 'My Company (San Francisco)' and 'Destination Location' (location_dest_id: 'CHIC1/Stock') belongs to another company.

Reply:
This is an error ar you account level for the stock picking configure the correct locations for it